config/initializers/session_store.rb

```diff
@@ -1,5 +1,14 @@
 # Be sure to restart your server when you modify this file.
 
 # Permanent cookie jar also uses 20 years
-Rails.application.config.session_store :cookie_store, key: '_accounts_session',
-                                                      expire_after: 20.years
+
+class SessionStoreCookieName
+  def self.to_s
+    environment_name = Rails.application.secrets.environment_name
+    "_accounts_session#{'_' + environment_name if environment_name != 'prodtutor'}"
+  end
+end
+
+Rails.application.config.session_store :cookie_store, key: SessionStoreCookieName.to_s,
+                                                      expire_after: 20.years,
+                                                      domain: :all
```

Comment:
`domain: :all` will need some QA testing, because I remember looking into this for the SSO cookie and it not working properly in all cases. But I don't remember what was wrong.